File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/AddPokeballModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import com.sfh.pokeRogueBot.model.enums.PokeBallType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class AddPokeballModifierItem extends ModifierItem {

    public static final String TARGET = "AddPokeballModifierType";

    private int count;
    private PokeBallType pokeballType;

}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/AddPokeballModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

import com.sfh.pokeRogueBot.model.enums.PokeBallType

class AddPokeballModifierItem : ModifierItem() {

    companion object {
        const val TARGET = "AddPokeballModifierType"
    }

    var count: Int = 0
    var pokeballType: PokeBallType? = null
}